.gitignore

```diff
@@ -31,3 +31,7 @@ bwapi/libReplayToolTest/TestData/Replay_Tool_Actual.exe
 bwapi/ipch/
 Release_Binary/Chaoslauncher/__log.txt
 Release_Binary/ExampleAIClient/Source/
+
+/bwapi/packages/*
+!/bwapi/packages/repositories.config

```

Comment:
Hmm, maybe that was bad, seems github lost all your comments. And I always thought the warnings about rebasing with conflicts were irrational. Well, won't do that again.